.github/ISSUE_TEMPLATE/issue.yml

```diff
@@ -0,0 +1,31 @@
+name: Issue
```

Comment:
This quite doesn't match the template 😅 Also can you change the file name to bug_report.yml?

Reply:
Changed the file name and updated the contents as per the example repository. Do Check if it's ok .